openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTermTab/GlossaryTermTab.component.tsx

```diff
@@ -162,6 +175,35 @@ const GlossaryTermTab = ({
             <span className="text-grey-muted">{t('label.no-description')}</span>
           ),
       },
+      {
+        title: t('label.reviewer'),
+        dataIndex: 'reviewers',
+        key: 'reviewers',
+        width: '33%',
+        render: (reviewers: EntityReference[]) => (
+          <OwnerLabel owners={reviewers} />
+        ),
+      },
+      {
+        title: t('label.synonym-plural'),
+        dataIndex: 'synonyms',
+        key: 'synonyms',
+        width: '33%',
+        render: (synonyms: string[]) => {
+          return (
+            <div className="d-flex flex-wrap">
+              {synonyms?.map((synonym: string) => (
+                <TagButton
+                  className="glossary-synonym-tag"
+                  key={synonym}
+                  label={synonym}
+                />
+              ))}
+              {synonyms?.length === 0 && <div>{NO_DATA_PLACEHOLDER}</div>}
+            </div>
+          );
```

Comment:
```suggestion
         {isEmpty(synonyms)?<div>{NO_DATA_PLACEHOLDER}</div>:
            <div className="d-flex flex-wrap">
              {synonyms.map((synonym: string) => (
                <TagButton
                  className="glossary-synonym-tag"
                  key={synonym}
                  label={synonym}
                />
              ))}
            </div>}
          )
```

Reply:
replaced the previous implementation with Lodash functions as recommended.